client/my-sites/plans/jetpack-plans/product-grid/index.tsx

```diff
@@ -61,8 +59,7 @@ const ProductGrid: React.FC< ProductsGridProps > = ( {
 	const currencyCode = useSelector( getCurrentUserCurrencyCode );
 	const currentPlanSlug =
 		useSelector( ( state ) => getSitePlan( state, siteId ) )?.product_slug || null;
-	const exPlatVariation =
-		useSelector( ( state ) => getVariationForUser( state, SWITCH_PLAN_SIDES_EXPERIMENT ) ) || '';
+	const exPlatVariation = null;
```

Comment:
This variable can be removed and `exPlatVariation === SWITCH_PLAN_SIDES_TREATMENT` below can be replaced with `false`.